### DISCUSSION TITLE
Feb 13 RaspOVOS GUI missing on device setup wifi option

### DISCUSSION THREAD

**Discussion by dcveteran** [OpenVoiceOS/ovos-buildroot#222] [General]
I have been working on some side projects and dove back into this today. Installed the Feb 13 raspOVOS GUI on a Pi4B+, the setup is missing the "on device setup" wifi option. I'll try a few older images, but if anyone happens to hop on today and knows the most recent stable version, please let me know.

**Comment by dcveteran** [comment]
I remembered I had a good working buildroot image from Jan 17, 2023 that worked well, so I am up and running on that again now.

I'm wondering though if there is there a repository of old buildroot images (like the raspovos ones).

I found this link for the current image, but can't find any of the older versions:
https://drive.google.com/drive/folders/113-zmx6ncoeLNsayseNxoaTlaAk1AfU2

**Comment by j1nx** [comment]
Can you upload the older buildroot image? I lost it.

will then upload it to downloads.openvoiceos.org

**Comment by dcveteran** [comment]
It's too big to put here, so here's a link to my Dropbox:
https://www.dropbox.com/scl/fi/wqmzsrglc993n0zcvg7f7/OpenVoiceOS_rpi4_64-gui_dev-202301171328.img.zip?rlkey=vb9u3ungaots4mm1k7pw88mkj&dl=0

Let me know when you have it so I can pull it back down.

**Comment by dcveteran** [comment]
Well, I spoke too soon on the Jan 17, 2023 buildroot image. The initial WiFi setup worked well but the touchscreen is inop and the system does not fully boot after a clean reboot or power off. I thought this one was working before for my hardware, maybe it was another image.

**Comment by builderjer** [comment]
There were issues with the "on device" wifi setup.  I don't recall exactly what they were, but that option was removed because of it.  The "mobile" setup works well though
